src/types.ts

```diff
@@ -137,3 +137,11 @@ export interface User {
 export interface RewardOptions {
   amount?: number;
 };
+
```

Comment:
could you add StoreItem and UserItem types?

Reply:
done!